File: Exercises_Python/1133.py
primeiro = int(input())
segundo = int(input())

numeros = [primeiro, segundo]

maior = max(numeros)
menor = min(numeros)

for c in range(menor+1,maior):
    if (c % 5 == 2) or (c % 5 == 3):
        print(c)

File: Bee-Crowd/Exercises_C++/1133.cpp
#include <iostream>
using namespace std;

int main() {

    int x,y,maior,menor;
    cin >> x;
    cin >> y;

    maior = (x + y + abs(x-y)) / 2;
    menor = (x + y - abs(x-y)) / 2;


    for (int c = menor+1; c < maior; c++) {
        if (c % 5 == 2 or c % 5 == 3) {
            cout << c << endl;
        }
    }
}